src/managers/ThreadManager.js

```diff
@@ -91,14 +91,19 @@ class ThreadManager extends BaseManager {
    *   .create({
    *     name: 'food-talk'
```

Comment:
missed a `,` here

Reply:
Thanks, seems that was always missing 🤦‍♂️